app/controllers/hyrax/contact_form_controller.rb

```diff
@@ -58,14 +59,12 @@ def create
         after_deliver
       else
         flash.now[:error] = 'Sorry, this message was not sent successfully. ' +
-                            @contact_form.errors.full_messages.map(&:to_s).join(", ") +
-                            "" + @captcha.error
+                            @contact_form.errors.full_messages.map(&:to_s).join(", ")
```

Comment:
Are we accounting for any captcha errors?

Reply:
I don't know... I'm just putting back what was there before that I removed accidentally.